src/pages/programs/program-solution-entity/program-solution-entity.ts

```diff
@@ -60,6 +62,68 @@ export class ProgramSolutionEntityPage {
       });
   }
 
+  getSubmissionArr() {
```

Comment:
Can we reuse this in programs and the institution's flow?

Reply:
same reason as the structure is different i am using two different functions to manipulate the data in each flows